scheduler/tasks.py

```diff
@@ -0,0 +1,69 @@
+import json
+import redis
+import requests
+
+from config.celery import app
+from celery.utils.log import get_task_logger
+from datetime import timedelta
+from django.conf import settings
+from django.utils import timezone
+
+from .models import ReminderSchedule
+
+
+logger = get_task_logger(__name__)
+r = redis.StrictRedis(host='localhost', port=6379)
+
+@app.task(ignore_result=True,)
+def check_for_scheduled_reminders():
+    logger.info("Checking for due reminders")
+
+    threshold = timezone.now() + timedelta(minutes=5)
+    yesterday = timezone.now() + timedelta(hours=23)
```

Comment:
Wouldn't `yesterday` be `timezone.now() - timedelta(hours=24)`?

Reply:
Yes! I'd fixed this but forgot to push it :facepalm: